fobi_custom/plugins/form_elements/fields/intercept/household_tenure/fobi_form_elements.py

```diff
@@ -9,7 +9,7 @@ class HouseholdTenurePlugin(FormFieldPlugin):
     """HouseholdTenurePlugin."""
 
     uid = "household_tenure"
-    name = "How many years have you lived at your current address?"
+    name = "What year did you move into your current address?"
```

Comment:
Should this be a dropdown field instead of an integer field?

Reply:
I think it works fine as an integer field, but we could definitely change it to a dropdown where the user selects from a set of bins. To me it seems like the cost incurred by that approach would be that custom analyses of the exported data would have to be limited in resolution by our preset bins. Do you think there's an upside to making it a select field that would offset that cost?

Reply:
I was actually imagining a dropdown with single values ranging from, say, 1919 - 2019. That would prevent people from entering values in the future or too far in the past to be possible. Now that I'm writing this out, though, I realize there's a better solution: setting a default minimum and maximum value on this field, like the one that's currently on the Number question. What do you think? Maybe this is just a nice-to-have.

Reply:
I like that idea a lot! I'll add it in. The minimum/maximum is enforced on the Fobi level, right? If so, then I can set the minimum to something like `1900` and set the maximum to the current year.

Reply:
I thought enforcing min/max for IntegerFields would be an out-of-the-box configuration in Fobi but it looks like it'll require looking through the source and copying their implementation, so I kicked this off to #161.